### PR TITLE
VZ-4767 multi-cluster logging

### PR DESCRIPTION
**Pull request by pfmackin**
Cleanup logging for Multicluster

1. Use VerrazzanoLogger
2. Don't return error to controller-runtime
3. Cleanup log message

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/controllers/clusters/vmc_controller.go [hunk 1]
```diff
@@ -155,7 +192,7 @@ func (r *VerrazzanoManagedClusterReconciler) syncServiceAccount(vmc *clustersv1a
 	// Does the VerrazzanoManagedCluster object contain the service account name?
 	saName := generateManagedResourceName(vmc.Name)
 	if vmc.Spec.ServiceAccount != saName {
-		r.log.Infof("Updating ServiceAccount from %q to %q", vmc.Spec.ServiceAccount, saName)
+		r.log.Oncef("Updating ServiceAccount from %q to %q", vmc.Spec.ServiceAccount, saName)
```
Comment (mgianatagh) [inline platform-operator/controllers/clusters/vmc_controller.go:195]: Do we have a guideline for using %s versus %q

Comment (pfmackin) [reply platform-operator/controllers/clusters/vmc_controller.go:195]: i didn't notice %q, it would be in the go fmt doc, let me look.

Comment (pfmackin) [reply platform-operator/controllers/clusters/vmc_controller.go:195]: this logs the string in a double quoted string

Comment (pfmackin) [reply platform-operator/controllers/clusters/vmc_controller.go:195]: https://pkg.go.dev/fmt

Comment (mgianatagh) [reply platform-operator/controllers/clusters/vmc_controller.go:195]: Right, my question was more aimed towards do we have guidelines of when to log strings enclosed in double quotes versus not quoted at all.  In almost all cases we do not double quote.  But if the content could include white space I can see the value of the double quotes.

**Review by mgianatagh — APPROVED** [review]

File: platform-operator/controllers/clusters/vmc_controller.go (hunk 1 above)
Comment (pfmackin) [reply platform-operator/controllers/clusters/vmc_controller.go:195]: that is a good point, i can add it to the guidelines.

**Pull request merged**